assets/l10n/app_en.arb

```diff
@@ -440,5 +440,13 @@
   "allMessagesPageTitle": "All messages",
   "@allMessagesPageTitle": {
     "description": "Title for the page of all messages"
+  },
+  "groupDMNotificationMessage": "{senderFullName} to you and {num, plural, =1{1 other} other{{num} others}}.",
+  "@groupDMNotificationMessage": {
+    "description": "Notification message for a group DM.",
+    "placeholders": {
+      "senderFullName": {"type": "String", "example": "Alice"},
+      "num": {"type": "int", "example": "4"}
```

Comment:
The name "num" isn't very helpful here as to which number the caller should pass. After all, the right answer involves subtracting 2 from the number it first lays hands on (which is the total number of users in the conversation).

Instead, `numOthers` would be a good name.

Reply:
Fixed. I have updated it with `numOthers`.